### PR TITLE
Update osu!talk listing and formatting

### PR DESCRIPTION
**Pull request by TPGPL**

### CODE REVIEW

File: wiki/osu!talk/en.md [hunk 1]
```diff
@@ -1,95 +1,110 @@
...
+| 19 | [Doomsday](https://osu.ppy.sh/users/18983) | Pro osu! player | [Link](https://www.youtube.com/watch?v=0C74QeEcn_4) |
 | 20 | [Tom94](https://osu.ppy.sh/users/1857058) | Developer | [Link](https://www.youtube.com/watch?v=ONnUrG4jrto) |
 | 21 | [Flanster](https://osu.ppy.sh/users/447818) | GMT | [Link](https://www.youtube.com/watch?v=nvGP5x9ZseM) |
 | 21.5 | [Blue Dragon](https://osu.ppy.sh/users/19048) | Mapper | [Link](https://puu.sh/cmOO3/a737a268da.mp3) |
-| 22 | [HappyStick](https://osu.ppy.sh/users/256802) | Pro Player | [Link](https://www.youtube.com/watch?v=zhAHOreuYp4) |
-| 23 | [Hayabusa](https://osu.ppy.sh/users/3104108) | Top UK Mania Player | [Link](https://www.youtube.com/watch?v=1C102Zzuyzg) |
-| 24 | [Kyonko Hiraza](https://osu.ppy.sh/users/444868) | Retired Pro Player | [Link](https://www.youtube.com/watch?v=6RhBqhhn9F0) |
-| 25 | [PortalLife](https://osu.ppy.sh/users/929134) | Player/Community Tournaments | [Link](https://www.youtube.com/watch?v=odGwuBwqcmc) |
-| 26 | [MillhioreF](https://osu.ppy.sh/users/941094) | Bagel Lord | [Link](https://www.youtube.com/watch?v=dO3kv8nutSI) |
```
Comment (cl8n) [inline wiki/osu!talk/en.md:47]: did these descriptions or the new ones come from anywhere? it feels like it was written this way on purpose

Comment (TPGPL) [reply wiki/osu!talk/en.md:47]: i couldn't find from where were the professions taken, so i replaced them with actual meaningful descriptions (based on `People` articles and video descriptions)

probably the context is in the videos themselves, basing on this comment from the `Bagel Lord` episode:
image

File: wiki/osu!talk/en.md [hunk 2]
```diff
@@ -1,95 +1,110 @@
...
+| 6 | osu! World Cup Grand Finals | [cptnXn](https://osu.ppy.sh/users/495272), [Doomsday](https://osu.ppy.sh/users/18983), [Loctav](https://osu.ppy.sh/users/71366), [Mr Color](https://osu.ppy.sh/users/116078), and [Tom94](https://osu.ppy.sh/users/1857058) | [Link](https://www.youtube.com/watch?v=bl63NpUz_T8) |
+| 7 | Guest difficulties vs single mapper set | [Doomsday](https://osu.ppy.sh/users/18983), [shARPII](https://osu.ppy.sh/users/776257), and [WubWoofWolf](https://osu.ppy.sh/users/39828) | [Link](https://www.youtube.com/watch?v=OMsHFRS662k) |
+| 8 | osu!next | [flyte](https://osu.ppy.sh/users/3103765), and [peppy](https://osu.ppy.sh/users/2) | [Link](https://www.youtube.com/watch?v=jBUNIDa427Q) |
+| 9 | Legends of the addicting circle clicking | [Charles445](https://osu.ppy.sh/users/85000), [Kyonko Hizara](https://osu.ppy.sh/users/444868), LoliFlan, and [p3n](https://osu.ppy.sh/users/123703) | [Link](https://www.youtube.com/watch?v=tIEuc4DK_nM) |
```
Comment (TicClick) [inline wiki/osu!talk/en.md:78]:
```suggestion
| 9 | Legends of the addicting circle clicking | [Charles445](https://osu.ppy.sh/users/85000), [Kyonko Hizara](https://osu.ppy.sh/users/444868), [Flanster](https://osu.ppy.sh/users/447818), and [p3n](https://osu.ppy.sh/users/123703) | [Link](https://www.youtube.com/watch?v=tIEuc4DK_nM) |
```

**Review by TicClick — COMMENTED** [review]
the titles most likely have arrived from Mediawiki, as for who put them there.. no idea. I enjoy the updated version more

File: wiki/osu!talk/en.md [hunk 3]
```diff
@@ -1,95 +1,110 @@
...
+| 41 | [Histoire](https://osu.ppy.sh/users/3801463) | Host of osu! Histy Championships | [Link](https://www.youtube.com/watch?v=3Q1ygMaAb7g) |
+| 42 | [jakads](https://osu.ppy.sh/users/259972) | Top osu!mania player | [Link](https://www.youtube.com/watch?v=MXx6oknK6c8) |
+| 43 | [Evening](https://osu.ppy.sh/users/2193881) | Resident Camellia mapper | [Link](https://www.youtube.com/watch?v=gHKzBz8hcoE) |
+| 44 | [pishifat](https://osu.ppy.sh/users/3178418) | QAT | [Link](https://www.youtube.com/watch?v=YseljuHjmLo) |
+| 45 | [Rafis](https://osu.ppy.sh/users/2558286) | Pro osu! player | [Link](https://www.youtube.com/watch?v=wKhuovIMa8k) |
+| 46 | [Spare](https://osu.ppy.sh/users/2204373) | Top UK osu! player | [Link](https://www.youtube.com/watch?v=MTWgwsIxPRc) |
+| 47 | [Helblinde](https://osu.ppy.sh/users/48053) | Featured artist | [Link](https://www.youtube.com/watch?v=cviwU4xkM-w) |
+| 48 | [MatsumotoRise](https://osu.ppy.sh/users/672726) | Pro osu! player | [Link](https://www.youtube.com/watch?v=8-3d2ZHw2O4) |
+| 49 | [Toy](https://osu.ppy.sh/users/2757689) | Pro osu! player | [Link](https://www.youtube.com/watch?v=lI8mIJLOu_k) |
```
Comment (TicClick) [inline wiki/osu!talk/en.md:64]: make the linter happy
```suggestion
| 49 | [Toy](https://osu.ppy.sh/users/2757689) | Pro osu! player | [Link](https://www.youtube.com/watch?v=lI8mIJLOu_k) |
```

**Review by TicClick — APPROVED** [review]

**Pull request merged**